src/golang/lib/repos/tests/operator.go

```diff
@@ -222,7 +222,7 @@ func (ts *TestSuite) TestOperator_GetUnusedCondaEnvNames() {
 	workflows := ts.seedWorkflowWithUser(1, userIDs)
 	workflowIDs := sampleWorkflowIDs(1, workflows)
 	dags := ts.seedDAGWithWorkflow(2, []uuid.UUID{workflowIDs[0], workflowIDs[0]})
-	historicalOp := ts.seedOperatorAndDAG(artifactID, dags[0].ID, users[0].ID, operator.FunctionType)
+	historicalOp := ts.seedOperatorAndDAGOperatorToArtifact(artifactID, dags[0].ID, operator.FunctionType)
```

Comment:
this is related to https://linear.app/aqueducthq/issue/ENG-2667/disable-the-random-behavior-of-seedoperatoranddag